app/Resources/views/default/index.html.twig

```diff
@@ -50,22 +50,25 @@
 <style>
     body { background: #F5F5F5; font: 18px/1.5 sans-serif; }
     h1, h2 { line-height: 1.2; margin: 0 0 .5em; }
-    h1 { font-size: 36px; }
+    h1 { font-size: 26px; }
```

Comment:
This value can only be `24` or `28` (we use the traditional typographic scale: 16, 18, 21, 24, 28, 32, 36, 48, 64, 72, 96, etc.)